core/src/main/resources/hudson/logging/LogRecorderManager/sidepanel.jelly

```diff
@@ -34,10 +34,10 @@ THE SOFTWARE.
       <l:task href="${rootURL}/manage" icon="symbol-settings" title="${%Manage Jenkins}"/>
       <l:task href="." icon="icon-clipboard icon-md" title="${%Log Recorders}">
         <l:isAdmin>
-          <l:task href="new" icon="icon-new-package icon-md" title="${%New Log Recorder}"/>
+          <l:task href="new" icon="symbol-add" title="${%New Log Recorder}"/>
         </l:isAdmin>
       </l:task>
-      <l:task href="all" icon="icon-notepad icon-md" title="${%All Log Messages}"/>
+      <l:task href="all" icon="icon-clipboard icon-md" title="${%All Log Messages}"/>
```

Comment:
This used to be different from the main entry (line 35). How about using the same icon as used on Manage Jenkins for System Log here on line 35?